src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorProjectChangePublisher.cs

```diff
@@ -103,7 +104,10 @@ internal void EnqueuePublish(ProjectSnapshot projectSnapshot)
         {
             // A race is not possible here because we use the main thread to synchronize the updates
```

Comment:
The "a race is not possible" followed by a lock is odd. Should probably expand the comment.